src/liballoc/rc.rs

```diff
@@ -588,7 +699,7 @@ impl<T: ?Sized + PartialEq> PartialEq for Rc<T> {
     ///
     /// let five = Rc::new(5);
     ///
-    /// five != Rc::new(5);
+    /// assert!(five != Rc::new(6));
```

Comment:
Just kidding, missed the `!=`. On a side note: https://github.com/rust-lang/rfcs/pull/1653